.circleci/config.yml

```diff
@@ -358,6 +358,9 @@ commands:
       config_branch:
         type: string
         default: *github_dispatch_config_branch
+      use_current_branch_name:
```

Comment:
Technically, we're not using the same branch, since we're pulling file from ddp-angular repo. Hence, the name is `use_current_branch_name` instead of `use_current_branch`.

Reply:
Hmm. I am not sure about this, Yufeng. Note that in the config repo the dispatch.yml files are already organized in separate folder by environment (dev/test/etc.). So going by branch AND folder seems to me adds confusion.
Will have to remember to create branch of same name in ddp-angular repo when all you are changing is something in pepper-backend .
The current setup (i.e., housing deployment configuration in ddp-angular/develop) is less than ideal; in Circle script we just treat as a place to store files. Perhaps best solution would be to create separate repo for configuration? Was discussed at time but seemed like too much for too little.
I am agreeing that current setup is not great, but not sure about this solution. Maybe a Slack discussion is in order?

Reply:
I welcome other solutions here. The trade-off is a bit more confusion, but it brings more safety to deployments. The cross-pollution from `develop` is undesirable. Note that this setup ONLY for when deploying branches named `test`, `staging`, and `production`. For those three branches, the branch in both `ddp-study-server` and `ddp-angular` should already be "in sync", if not we probably have other issues.